modelci/__init__.py

```diff
@@ -6,10 +6,9 @@
 Date: 9/19/2020
 """
 from . import data_engine
-from . import hub
 from . import metrics
 from . import monitor
 from . import types
 from . import utils
 
-__all__ = ['data_engine', 'hub', 'metrics', 'monitor', 'types', 'utils']
+__all__ = ['data_engine', 'metrics', 'monitor', 'types', 'utils']
```

Comment:
why remove hub?

Reply:
`hub` imports a lot of `PyTorch` and `TensorFlow` dependency. It is better not import at the beginning. Otherwise, importing `modelci` will have a lot of dependencies and cost a lot of time.